charon/src/translate/translate_traits.rs

```diff
@@ -389,15 +389,19 @@ impl<'tcx, 'ctx> TranslateCtx<'tcx, 'ctx> {
                 generic_clauses
             );
         }
-
+        let item_meta = bt_ctx.t_ctx.translate_item_meta_from_rid(rust_id);
+        if item_meta.opaque {
+            let ctx = bt_ctx.into_fmt();
+            bt_ctx.t_ctx.errors.session.span_warn(item_meta.span, format!("The `aeneas::opaque` or `charon::opaque` attribute currently has no effect on trait declarations and will be ignored. \n- Declaration name: {}\n", name.fmt_with_ctx(&ctx)))
```

Comment:
```suggestion
            bt_ctx.t_ctx.errors.session.span_warn(item_meta.span, format!("The `aeneas::opaque` or `charon::opaque` attribute currently has no effect on trait declarations and will be ignored.\n- Declaration name: {}\n", name.fmt_with_ctx(&ctx)))
```
This is a super long line, could you break it?
(note that I'm surprised that the formatter doesn't do it on itself)